README.md

```diff
@@ -1,118 +1,15 @@
-
-#### Host a domain server and Hagall under the same domain name
-1. Change `example.com` to your domain name in `docker-compose-allinone.yml`, trailing slash is essential in `HAGALL_PUBLIC_ENDPOINT`.
-2. If you have Hagall running using this [docker-compose file](https://github.com/aukilabs/hagall/blob/main/docker-compose.yml), make sure to stop your Hagall server first, e.g. `docker compose -f hagall-docker-compose-file-path.yaml down`, in order to release port 443 and 80.
-3. Run `docker compose -f docker-compose-allinone.yml up -d` to start Hagall and the domain server under the same domain. Hagall will be hosted at `https://your-domain-name/hagall/` and the domain server at `https://your-domain-name/domains/`
-4. You will always need to specify `-f docker-compose-allinone.yml` when you run any Docker Compose command like `up`, `pull`, `down` and `stop`.
-
-#### Upgrading
-
-You can do the same steps as for Docker, but if you're not already running a domain server or you have modified the `docker-compose.yml` file recently and want to deploy the changes, you can navigate to the folder where you have your `docker-compose.yml` file and then run `docker compose pull` followed by `docker compose down` and `docker compose up -d`.
-
-Note that the `docker compose pull` command will also upgrade the other containers defined in `docker-compose.yml` such as the nginx proxy and the Let's Encrypt helper.
-
-### Binary
-
-> **_NOTE:_** If you choose to use this deployment method, you need to set up your own HTTPS web server or reverse proxy with an SSL certificate. Domain server listens for incoming connections on port 4000 by default, but this is changeable, see the configuration options above.
-
-#### Currently supported platforms
-
-We build pre-compiled binaries for these operating systems and architectures:
-
-- Windows x86, x86_64
-- macOS x86_64, ARM64 (Apple Silicon)
-- Linux x86, x86_64, ARM, ARM64
-- FreeBSD x86, x86_64
-- Solaris x86_64
-
-> **_NOTE:_** Auki Labs doesn't test all of these platforms actively. Windows, FreeBSD and Solaris builds are currently experimental. We don't guarantee that everything works but feel free to reach out with your test results.
-
-1. Download the latest version of the domain server from [GitHub](https://github.com/aukilabs/domain-server/releases).
-2. Run it with `./ds --public-url=https://domains.example.com --registration-credentials=xxx`
-3. Expose it using your own reverse proxy with an SSL certificate.
 
-We recommend you use a daemon manager such as systemd, launchd, SysV Init, runit or Supervisord to make sure your domain server stays running at all times.
+The Domain Server is responsible for storing poses of portals in domains and spatial domain data such as scene reconstructions and occupancy maps.
 
-### Docker
+Every Hagall server needs a unique wallet to participate in the [posemesh economy](https://www.posemesh.org/#economy).
```

Comment:
This should be `Every domain server...` I guess.

Reply:
Good catch. Please check again :)